tensorforce/models/ppo_model.py

```diff
@@ -36,23 +38,24 @@ class PPOModel(PolicyGradientModel):
     default_config = dict(
         entropy_penalty=0.01,
         loss_clipping=0.2,  # Trust region clipping
-        epochs=10,  # Number of training epochs for SGD,
```

Comment:
I think we are using the configure optimiser, not only SGD so I called it the optimiser instead